server/json/user1-institutions.json

```diff
@@ -6,47 +6,53 @@
       "status": {
         "code": 0,
         "message": ""
-      }
+      },
+      "currentSubmission": 0
```

Comment:
We don't use the timestamp at all for logic, it is just there for info purposes. This way we don't have to think about time or syncing or timezones or anything. Just a backend service that increments the submission count.